Lib/test/test_zipimport.py

```diff
@@ -776,6 +776,16 @@ def testLargestPossibleComment(self):
         files = {TESTMOD + ".py": (NOW, test_src)}
         self.doTest(".py", files, TESTMOD, comment=b"c" * ((1 << 16) - 1))
 
+    def testZip64(self):
+        # This is the simplest way to make zipfile generate the zip64 EOCD block
```

Comment:
```suggestion
        # This is the simplest way to make zipfile generate the zip64 EOCD block.
```

Should this be broken out into a utility function? Or have a separate `TestCase` for ZIP64 where this is part of the `setUp()` method?